Makefile

```diff
@@ -103,6 +104,10 @@ emulator:
 	$(call print-target)
 	./emulator/setup.sh
 
+man:
+	$(call print-target)
+	go-md2man -in README.md -out magellan.1
+
```

Comment:
Since this relies on a program installed in the `inst` recipe, we'll want to add it as a prerequisite. Also in following with using `GOPATH` to run programs installed this way, we should apply it here as well:

```diff
diff --git a/Makefile b/Makefile
index f787545..29aa177 100644
--- a/Makefile
+++ b/Makefile
@@ -104,9 +104,10 @@ emulator:
        $(call print-target)
        ./emulator/setup.sh
 
-man:
+.PHONY: man
+man: inst
        $(call print-target)
-       go-md2man -in README.md -out magellan.1
+       $(GOPATH)/bin/go-md2man -in README.md -out magellan.1
 
 define print-target
     @printf "Executing target: \033[36m$@\033[0m\n"
```

Reply:
Should we have `.PHONY` here since this produces a file?

Reply:
Ah, yes that is a good point. I think a good way to do this is to have a `magellan.1` target so Make checks if it has been built, then have a phony target so the user can call it explicitly to build the man page. E.g:

```make
magellan.1: README.md
        $(GOPATH)/bin/go-md2man -in $< -out $@

.PHONY: man
man:
        $(call print-target)
        $(MAKE) -f $(firstword $(MAKEFILE_LIST)) magellan.1
```

Since you call `print-target` being called, we call `$(MAKE)` in the recipe. If we didn't have this, the `man` target would have `magellan/1` as its prerequisite.

Calling `man` first time:

```
$ make man
Executing target: man
make -f Makefile magellan.1
make[1]: Entering directory '/tmp/magellan'
/tmp/tmp.qqpuUX0YNN/bin/go-md2man -in README.md -out magellan.1
make[1]: Leaving directory '/tmp/magellan'
```
Calling `man` after it has been built:
```
$ make man
Executing target: man
make -f Makefile magellan.1
make[1]: Entering directory '/tmp/magellan'
make[1]: 'magellan.1' is up to date.
make[1]: Leaving directory '/tmp/magellan'
```

Reply:
You might want to add `inst` as a prerequisite to `magellan.1` since `go-md2man` is needed. `$<` only returns the first prerequisite, so the command should still work fine.

Reply:
It looks like upon further testing that since `inst` always runs when invoking `magellan.1`, `go-md2man` will always run. I think if I was nitpicking, I would want the latter only to run if it needs to be rebuilt but this is fine for the scope of this PR so I'll resolve this.